bach/bach/series/series_datetime.py

```diff
@@ -39,6 +42,53 @@ def sql_format(self, format_str: str) -> SeriesString:
         return str_series
 
 
+class TimeDeltaOperation(DateTimeOperation):
```

Comment:
nice to have this!

Reply:
Let's match the case of the Series
```suggestion
class TimedeltaOperation(DateTimeOperation):
```